recipes/correct-ts-specifiers/src/fixtures/e2e/.gitignore

```diff
@@ -0,0 +1 @@
+!node_modules
```

Comment:
maybe add a comment to explain why we want to keep that

Reply:
```suggestion
!node_modules # within this directory, these are also fixtures
```